### PR TITLE
central utah lidar page

### PR DESCRIPTION
**Pull request by rkelson**
Pull request description
------------------------

### CODE REVIEW

File: data/elevation-and-terrain/2018-lidar-central-utah/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  margin: 0 auto;
+}
+</style>
+<div id="logo">
+  <img src="{{ "/images/lidar_central_utah_2018.png" | prepend: site.baseurl }}" alt="Sample" />
+</div>
+
+[![Project area map]({{"/images/lidar_central_utah_2018_coverage_sm.png" | prepend:site.baseurl}} "click for map")]({{"/images/lidar_central_utah_2018_coverage.png" | prepend:site.baseurl}}){:.inline-text-right}
+
+During 2018 AGRC, the Utah Division of Emergency Management, the Natural Resource Conservation Service (NRCS), and the Utah Geological Survey (UGS) [~3,759 square miles]({{ "/images/lidar_central_utah_2018_coverage.png/" | prepend: site.baseurl }}) of 2 and 8 points per meter Quality Level 1 & 2 LiDAR for areas in Central Utah. The .5 and 1 meter resolution bare earth DEMs and first-return/highest-hit DSMs in .tif format have a 10.0cm vertical RMSE accuracy and are available for download. The LAS point clouds, classified to ground, are also available [by request](mailto:rkelson@utah.gov). This elevation data was collected between April and October, 2018 and has a UTM NAD83 (2011) zone 12 north meters NAVD88(GEOID12) projection.
```
Comment (steveoh) [inline data/elevation-and-terrain/2018-lidar-central-utah/index.md]: same mailto removal is necessary

File: data/elevation-and-terrain/2018-lidar-central-utah/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  <li>
+    <i class="fa fa-download"></i> Download project <a href="https://storage.googleapis.com/state-of-utah-sgid-downloads/lidar/central-utah-2018/CentralUtah_2018_Reports.zip" target="_blank">Reports</a> and
+      <a href="https://storage.googleapis.com/state-of-utah-sgid-downloads/lidar/central-utah-2018/CentralUtah_2018_Metadata.zip" target="_blank">Metadata</a>
+  </li>
+  <li>
+    <i class="fa fa-download"></i> Download <a href="https://storage.googleapis.com/state-of-utah-sgid-downloads/lidar/central-utah-2018/CentralUtah_2018_shps.zip" target="_blank">shapefiles</a> of project area, tile indices, and breaklines
+  </li>
+</ul>
+
+The naming convention for the tiles are based off the [U.S. National Grid (USNG)]( http://www.fgdc.gov/usng/how-to-read-usng/index_html).
```
Comment (steveoh) [inline data/elevation-and-terrain/2018-lidar-central-utah/index.md]:
```suggestion
The naming convention for the tiles are based off the [U.S. National Grid (USNG)](https://www.fgdc.gov/usng/how-to-read-usng/index_html).
```

File: data/elevation-and-terrain/2018-lidar-central-utah/index.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+This elevation data has a UTM NAD83 (2011) zone 12 north meters NAVD88(GEOID12) projection
+
+If you need assistance contact Rick Kelson at [rkelson@utah.gov](mailto:rkelson@utah.gov)
```
Comment (steveoh) [inline data/elevation-and-terrain/2018-lidar-central-utah/index.md]: remove mailto link

File: data/elevation-and-terrain/2018-lidar-central-utah/index.md [hunk 4]
```diff
@@ -43,8 +45,8 @@ During 2018 AGRC, the Utah Division of Emergency Management, the Natural Resourc
   </li>
 </ul>
 
-The naming convention for the tiles are based off the [U.S. National Grid (USNG)](https://www.fgdc.gov/usng/how-to-read-usng/index_html).
+The naming convention for the tiles are based off the [U.S. National Grid (USNG)]( https://www.fgdc.gov/usng/how-to-read-usng/index_html).
```
Comment (steveoh) [inline data/elevation-and-terrain/2018-lidar-central-utah/index.md]: why did you add back the space?

**Comment by rkelson** [comment]
they are copied and pasted from my original files and I failed to notice
the space.

Rick Kelson
Utah AGRC
801.538.3237
gis.utah.gov

**Comment by steveoh** [comment]
still needs image compression but LGTM

**Comment by rkelson** [comment]
the images are the same size as the Kanab images that you already approved
and merged so what is the problem?

Rick Kelson
Utah AGRC
801.538.3237
gis.utah.gov

**Comment by steveoh** [comment]
I compressed them afterwards. There is quite a bit to squeeze out of them.

https://github.com/agrc/gis.utah.gov/commit/95b8e2b145c77d50ae86bf79c054fcf7f1387ac6

**Comment by steveoh** [comment]
would you like this published?

**Comment by rkelson** [comment]
yes

Rick Kelson
Utah AGRC
801.538.3237
gis.utah.gov
